### PR TITLE
Add clients for mixed file ops and mixed smart contract ops tests for reconnect

### LINKED ISSUE #1342: Reconnect test scenarios for Files
**Summary**
Add reconnect test scenario in which the following file operations are done during and after reconnect
1. Files are created
2. Files are Updated
3. Files are Appended
Validate if the reconnect is successful with the file operations.

### LINKED ISSUE #1343: Reconnect test scenarios for SmartContracts
**Summary**
Add a test that creates and executes SmartContracts during reconnect and after the nodes are successfully reconnected.

**Possible resolution**
Lorem ipsum dolor sit amet.

### PR DESCRIPTION
**Pull request by Neeharika-Sompalli**
**Related issue(s)**:
Closes #1342 
Closes #1343 

Related PR in regression : 

**Summary of the change**:
1. Moved the performance suites to their respective packages for clarity
2. Added `MixedSmartContractOpsLoadTest` and  `MixedFileOpsLoadTest` for  submitting mixed transactions during reconnect

Passed tests : https://hedera-hashgraph.slack.com/archives/C018CBQBTGB/p1620073200063000

### CODE REVIEW

**Review by tinker-michaelj — APPROVED** [review]
LGTM!

File: test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/LoadTest.java [hunk 1]
```diff
@@ -59,6 +67,10 @@
 	public static OptionalInt totalTestTokenAccounts = OptionalInt.empty();
 	public static OptionalInt memoLength = OptionalInt.of(DEFAULT_MEMO_LENGTH);
 
+	protected final ResponseCodeEnum[] standardPermissiblePrechecks = new ResponseCodeEnum[] {
+			OK, BUSY, DUPLICATE_TRANSACTION, PLATFORM_TRANSACTION_NOT_CREATED
```
Comment (ljianghedera) [inline test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/LoadTest.java:71]: Should `PLATFORM_NOT_ACTIVE` be one of them?

File: test-clients/src/main/java/com/hedera/services/bdd/suites/perf/file/MixedFileOpsLoadTest.java [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+						.payingWith(GENESIS)
+						.hasAnyPrecheck()
+						.deferStatusResolution(),
+				fileAppend(targetFile)
+						.content("dummy")
+						.hasAnyPrecheck()
+						.payingWith(GENESIS)
+						.fee(ONE_HUNDRED_HBARS)
+						.logging()
+						.deferStatusResolution()
```
Comment (ljianghedera) [inline test-clients/src/main/java/com/hedera/services/bdd/suites/perf/file/MixedFileOpsLoadTest.java:91]: nit: These two operations will operate on the same file object.  Is this intended to be a hotspot perf testing?

**Review by anighanta — DISMISSED** [review]

File: test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/LoadTest.java (hunk 1 above)
Comment (Neeharika-Sompalli) [reply test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/LoadTest.java:71]: As `PLATFORM_NOT_ACTIVE` is not a permissible precheck, and the test should be stopped for that status, it is not added.

File: test-clients/src/main/java/com/hedera/services/bdd/suites/perf/file/MixedFileOpsLoadTest.java (hunk 2 above)
Comment (Neeharika-Sompalli) [reply test-clients/src/main/java/com/hedera/services/bdd/suites/perf/file/MixedFileOpsLoadTest.java:91]: @ljianghedera  This is not intended to be for performance testing though it extends LoadTest. This test is supposed to submit  file operations and modify few files during reconnect. As the files are blobs I believe they will create new blobs if modified.

File: test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/LoadTest.java (hunk 1 above)
Comment (ljianghedera) [reply test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/LoadTest.java:71]: I see your point. I was thinking there might be two scenarios that `PLATFORM_NOT_ACTIVE` might be a permissible precheck: 1) The client comes up slightly earlier than server node. If the client can retry several times, the test can still be successful; 2) The server node may be temporarily unavailable due to network glitch, the retry will be still helpful.

File: test-clients/src/main/java/com/hedera/services/bdd/suites/perf/file/MixedFileOpsLoadTest.java (hunk 2 above)
Comment (ljianghedera) [reply test-clients/src/main/java/com/hedera/services/bdd/suites/perf/file/MixedFileOpsLoadTest.java:91]: Got it.

**Review by ljianghedera — DISMISSED** [review]
LGTM overall.

File: test-clients/src/main/java/com/hedera/services/bdd/suites/perf/file/MixedFileOpsLoadTest.java [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+		final AtomicInteger submittedSoFar = new AtomicInteger(0);
+		final byte[] newContents = TxnUtils.randomUtf8Bytes(TxnUtils.BYTES_4K);
+		String initialContent = "The initial contents!";
+		String targetFile = "targetFile";
+
+		Supplier<HapiSpecOperation[]> mixedFileOpsBurst = () -> new HapiSpecOperation[] {
+				fileCreate(targetFile + submittedSoFar.getAndIncrement()).contents(initialContent),
+				fileUpdate(targetFile)
+						.fee(ONE_HUNDRED_HBARS)
+						.contents(newContents)
```
Comment (qnswirlds) [inline test-clients/src/main/java/com/hedera/services/bdd/suites/perf/file/MixedFileOpsLoadTest.java:80]: nit: even though `newContents` are random bytes, they don't change over the course of this load test. If your intention was to have them change, `TxnUtils.randomUtf8Bytes(TxnUtils.BYTES_4K)` must be moved here.

**Review by qnswirlds — DISMISSED** [review]
Overall LGTM

**Review by abhishek-hedera — APPROVED** [review]

File: test-clients/src/main/java/com/hedera/services/bdd/suites/perf/file/MixedFileOpsLoadTest.java (hunk 3 above)
Comment (Neeharika-Sompalli) [reply test-clients/src/main/java/com/hedera/services/bdd/suites/perf/file/MixedFileOpsLoadTest.java]: Fixed. Thanks @qnswirlds !

File: test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/LoadTest.java (hunk 1 above)
Comment (Neeharika-Sompalli) [reply test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/LoadTest.java:71]: @ljianghedera Got it. In my opinion for 
1. we need to use JRS setting to make sure clients start after server as this is Engineering testing
2. Though this is a a valid case, this case should not happen mostly, unless we are simulating Restart/Reconnect and trying to submit transactions to the node that is being restarted.

**Review by anighanta — APPROVED** [review]

**Review by abhishek-hedera — APPROVED** [review]

**Pull request merged**